### PR TITLE
ci(SW-27141): fix package creation

### PR DESCRIPTION
**Pull request by PascalThesing**
### 1. Why is this change necessary?
Currently, the script for creating the packages if broken.

### 2. What does this change do, exactly?
It fixes the scripts.

### CODE REVIEW

File: .github/workflows/create-release.yml [hunk 1]
```diff
@@ -48,16 +47,17 @@ jobs:
...
+                    composer install --no-dev -d recovery/common/
 
             -   name: Replace placeholder
                 run: |
-                    sed -i 's=___VERSION___=${{ github.ref_name }}=g' ./engine/Shopware/Kernel.php
-                    sed -i 's=___VERSION_TEXT___= =g' ./engine/Shopware/Kernel.php
-                    sed -i 's=___REVISION___=${{ env.REVISION }}=g' ./engine/Shopware/Kernel.php
-                    sed -i 's=___VERSION___=${{ github.ref_name }}=g' ./recovery/install/data/version
-                    sed -i 's=___VERSION_TEXT___= =g' ./recovery/install/data/version
+                    ls -la
```
Comment (mitelg) [inline .github/workflows/create-release.yml]: this was only for debug, please remove

File: .github/workflows/create-release.yml [hunk 2]
```diff
@@ -66,25 +66,27 @@ jobs:
...
                 run: |
-                    mysql -u root -proot -P ${{ env.DB_PORT }} -h ${{ env.DB_HOST }}  ${{ env.DB_NAME }} < ./_sql/install/latest.sql
+                    mysql -u root -proot -P ${{ env.DB_PORT }} -h ${{ env.DB_HOST }}  ${{ env.DB_NAME }} < _sql/install/latest.sql
 
             -   name: Generate new sql
                 run: |
                     make config
                     php bin/console sw:migrations:migrate --mode=install
                     php bin/console sw:snippets:to:sql ./recovery/install/data/sql/snippets.sql --force --include-default-plugins --update=false
+                    git status
```
Comment (mitelg) [inline .github/workflows/create-release.yml]: should also be removed

File: .github/workflows/create-release.yml [hunk 3]
```diff
@@ -66,25 +66,27 @@ jobs:
...
 
             -   name: Save sql
                 run: |
-                    mysqldump --column-statistics=0 --quick  -u root -proot -P ${{ env.DB_PORT }} -h ${{ env.DB_HOST }}  ${{ env.DB_NAME }} > ./recovery/install/data/sql/install.sql
+                    mysqldump --column-statistics=0 --quick  -u root -proot -P ${{ env.DB_PORT }} -h ${{ env.DB_HOST }}  ${{ env.DB_NAME }} > recovery/install/data/sql/install.sql
+                    git status
```
Comment (mitelg) [inline .github/workflows/create-release.yml]: should also be removed

File: .github/workflows/create-release.yml [hunk 4]
```diff
@@ -96,10 +98,20 @@ jobs:
...
+                run: |
+                    git config --global user.name "shopware5"
+                    git config --global user.email "<>"
+
             -   name: Create Zip
                 run: |
-                    git archive -v -o ${{ env.PACKAGE_PREFIX }}.zip --add-file=.htaccess --add-file=config.php --add-file=./recovery/install/data/sql/install.sql --add-file=./engine/Shopware/Components/Check/Data/Files.md5sums --add-file=./engine/Shopware/Components/HttpClient/cacert.pem HEAD
-                    zip -r ${{ env.PACKAGE_PREFIX }}.zip ./vendor ./recovery/common/vendor
+                    rm -rf _sql
+                    git status
```
Comment (mitelg) [inline .github/workflows/create-release.yml]: remove

File: .github/workflows/create-release.yml [hunk 5]
```diff
@@ -96,10 +98,20 @@ jobs:
...
+                    git add .
+                    git commit -m "Create release package"
+                    git status
```
Comment (mitelg) [inline .github/workflows/create-release.yml]: remove

File: .github/workflows/create-update.yml [hunk 6]
```diff
@@ -91,10 +92,20 @@ jobs:
...
+                run: |
+                    git config --global user.name "shopware5"
+                    git config --global user.email "<>"
+
             -   name: Create Zip
                 run: |
-                    git archive -v -o ${{ env.PACKAGE_PREFIX }}.zip --add-file=${{ env.UPDATE_ASSET_DIR }}version --add-file=${{ env.UPDATE_ASSET_DIR }}cleanup.txt --add-file=${{ env.UPDATE_ASSET_DIR }}snippets.sql --add-file=./engine/Shopware/Components/Check/Data/Files.md5sums --add-file=./engine/Shopware/Components/HttpClient/cacert.pem HEAD
-                    zip -r ${{ env.PACKAGE_PREFIX }}.zip ./vendor ./recovery/common/vendor ${{ env.UPDATE_ASSET_DIR }}
+                    rm -rf _sql
+                    git status
```
Comment (mitelg) [inline .github/workflows/create-update.yml]: remove

File: .github/workflows/create-update.yml [hunk 7]
```diff
@@ -91,10 +92,20 @@ jobs:
...
+                    git add .
+                    git commit -m "Create release package"
+                    git status
```
Comment (mitelg) [inline .github/workflows/create-update.yml]: remove

**Review by mitelg — CHANGES_REQUESTED** [review]

**Comment by mitelg** [comment]
is there a way to make sure, that update and install package have the same revision (time stamp)?

**Review by mitelg — APPROVED** [review]

**Review by DennisGarding — APPROVED** [review]

**Pull request merged**